pages/index.tsx

```diff
@@ -19,21 +34,136 @@ export default function Home(): ReactElement {
         <title>{t('title')}</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-
       <Header />
       <Main />
       <Footer />
+      {console.dir({ claimData })} {/* @TODO: Remove. For development purposes only. */}
     </Container>
   )
 }
 
+export interface QueryParams {
+  user_key: string
+  uniqueNumber: string
+}
+
+function buildApiUrl(url: string, queryParams: QueryParams) {
+  const apiUrl = new URL(url)
+
+  for (const key in queryParams) {
+    apiUrl.searchParams.append(key, queryParams[key as 'user_key' | 'uniqueNumber'])
+  }
+
+  return apiUrl.toString()
+}
+
 export const getServerSideProps: GetServerSideProps = async ({ req, locale }) => {
   const isProd = process.env.NODE_ENV === 'production'
   const logger = isProd ? pino({}) : pino({ prettyPrint: true })
   logger.info(req)
 
+  /*
+   * Load environment variables to be used for authentication & API calls.
+   * @TODO: Handle error case where env vars are null or undefined.
```

Comment:
I removed the previous non-null assertions because it was failing the eslint [no-non-null-assertion](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-non-null-assertion.md) rule, which is [recommended](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin). Here's a [writeup](https://medium.com/@tar.viturawong/a-note-on-typescript-non-null-assertion-operator-bad8fbe62bba) on what the non-null assertions were doing, whiche was essentially allowing typescript to ignore the fact that these might actually be undefined. Instead of ignoring, we should actively handle that error case (in a future ticket).

For now, I'm doing some [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) and just setting to empty string if the env var is undefined/null.

Reply:
Great, thank you!